opentelemetry-api/setup.py

```diff
@@ -39,8 +44,15 @@
     ],
     extras_require={},
     license="Apache-2.0",
-    packages=find_packages(exclude=("examples", "tests",)),
-    namespace_packages=[],
-    url="https://github.com/open-telemetry/opentelemetry-python/opentelemetry-api",
+    packages=[
```

Comment:
It seems there exists a setuptools function for this: https://setuptools.readthedocs.io/en/latest/setuptools.html#find-namespace-packages (I wonder why it is not mentioned in the packaging guide).

Reply:
Will try. Thanks!